app/models/group.js

```diff
@@ -224,5 +240,51 @@ export function addModel(dbAdapter) {
 
       return [timeline];
     }
+
+    /**
+     * Blocks user in the group. adminId is the id of the admin who blocking the
```

Comment:
Which part of the system that the user performing the action has rights to perform it?

Reply:
I usually try to check the rights only in controllers, not in models. This is not always the case, but in new methods it usually is. So, fot this method the rights are checked here: https://github.com/FreeFeed/freefeed-server/pull/577/files#diff-b9ba665c86ff402d057c38cd02119f0f8c13daa404e58f9e8c4321dacbd903f8R93-R99